packages/react-native-codegen/src/generators/RNCodegen.js

```diff
@@ -42,6 +42,33 @@ const generateViewConfigJs = require('./components/GenerateViewConfigJs.js');
 const path = require('path');
 const schemaValidator = require('../SchemaValidator.js');
 
+const ALL_GENERATORS = {
```

Comment:
Can we do that `ALL_GENERATORS` is the merge of `SCHEMA_GENERATORS` U `LIBRARY_GENERATORS` U any generator not included above? 

In that way if we add a generator in SCHEMA or LIBRARY we don't have to list it twice (and possibly forget to add it to the ALL generators).

WDYT?

Reply:
They are different. `SCHEMA_GENERATORS` and `LIBRARY_GENERATORS` return generators by category name, `ALL_GENERATORS` returns generators by its own name. They can replace or be calculated from each other.

Reply:
I would rather avoid changing the structure of `SCHEMA_GENERATORS` and `LIBRARY_GENERATORS` because those are used in both iOS and Android codegen and changing them could have ripple effects in various parts of the Codebase.

I verified that the only generator `ALL_GENERATORS` has more than the union of the other is the `generateViewConfigJs`.  So in theory, we could reduce the union of `SCHEMA_GEN` and `LIBRARY_GEN` into the `ALL_GEN` dictionary... but I don't know if we can extract the key from the name of the generator.

I'd go for the simplest solution for now, which is what you already implemented here, so won't need to change anything.